src/settings/settings.js

```diff
@@ -0,0 +1,25 @@
+import "./settings.scss";
+import bodyStyles from "../styles.scss";
+import {
+  setDataInStorage,
+  getDataFromStorage,
+} from "../utils/localStorage/localStorage.js";
+
+const SAVE_BUTTON = document.querySelector(".section__button--save");
+const INPUT = document.querySelector(".section__input--playerName");
+
+const onbuttonClick = () => {
+  setDataInStorage("playerName", INPUT.value);
+};
+
+function getPlayerNameIfExists() {
```

Comment:
Change it to `setPlayerName` - the result in either way will always be having a default name if empty (`Player`) or a custom name created in the past.